src/app/classify.rs

```diff
@@ -3,22 +3,36 @@ use http;
 
 use proxy::http::classify;
 
-#[derive(Clone, Debug, Default)]
-pub struct Classify {}
+pub use proxy::http::classify::insert;
 
 #[derive(Clone, Debug, Default)]
-pub struct ClassifyResponse {}
+pub struct Request {}
```

Comment:
Nit: This can also be declared as just
```suggestion
pub struct Request;
```
and then you can construct it by just saying `Request`, rather than having to say `Request {}`.

TIOLI.

Reply:
In the next PR this gets more stuff

Reply:
ah okay, carry on then :)